conf/vertebrates/next_production_reg_conf.pl

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Release Coordinator, please update this file before starting every release
+# and check the changes back into GIT for everyone's benefit.
+
+# Things that normally need updating are:
+#
+# 1. Release number
+# 2. Check the name prefix of all databases
+# 3. Possibly add entries for core databases that are still on genebuilders' servers
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Registry;
+use Bio::EnsEMBL::Compara::Utils::Registry;
+
+my $next_release = $ENV{'CURR_ENSEMBL_RELEASE'};
+my $curr_release = $next_release - 1;
+my $prev_release = $curr_release - 1;
+
+my $prev_db_suffix = Bio::EnsEMBL::Compara::Utils::Registry::PREVIOUS_DATABASE_SUFFIX;
+
+# ---------------------- SPECIES TO UPDATE OUT OF THE RELEASE ------------------
+
+my $updated_core_dbs = {
+    'mus_musculus' => [ 'mysql-ens-genebuild-prod-2', 'thibaut_mus_musculus_havana_39' ],
+};
+
+my $prev_core_dbs = {
+    "mus_musculus$prev_db_suffix" => [ 'mysql-ens-vertannot-staging', 'mus_musculus_core_102_38' ],
```

Comment:
The aliases are created that way by the registry (`load_registry_from_url()` and `load_registry_from_db()`), so that is the easiest way I could think of to define the new cores and be able to use both keys and hashes to replace them.
And yes, I think only the Fantastic 4 are meant to be involved in this out-of-the-release process.

Reply:
Those are the species that would have a higher impact in our production, yes, but I wasn't sure if all those should be run out of the release.

Reply:
if we can I think it is better to run them out of the release